### DISCUSSION TITLE
Logbook or other entry for User when armed / disarmed

### DISCUSSION THREAD

**Discussion by evolv30101** [Dilbert66/esphome-dsckeybus#146] [Q&A]
Trying to find an log entry somewhere in home assistant for which user armed and/or disarmed the system to be used for notifications.  
I remember seeing it on an old build using the main branch somewhere in the logbook but cant seem to find it now that i switched over to the dev branch.  I do have the user slots setup in the esp config with names.   

Idea being to send out a notification when the system is armed/disarmed with the matching user's name (rather than a slot number) for easy identification.

**Comment by Dilbert66** [comment]
The "evt" (event) field has those details.

**Comment by evolv30101** [comment]
EDIT:  Nevermind....im an idiot :)

I found it under the evt sensor.   

Thanks
